### PR TITLE
Add support for Pinscape Lite

### PR DESCRIPTION
**Pull request by MikePinball**
Add native support for Pinscape Lite expansion board with "type = 3". 

Before posting new build the URL for Pinscape Lite needs to be changed to match the official announcement thread on VPforums so a little bit of coordination is needed.

### CODE REVIEW

File: PinscapeConfigTool/html/Configure.htm [hunk 1]
```diff
@@ -6167,15 +6249,23 @@ <h1>
         function mainBoardHoverName(pin)
         {
             // if there's an out port alias, return the jumper entry
-            var alias = outPortAlias[pin];
```
Comment (mjrgh) [inline PinscapeConfigTool/html/Configure.htm:6170]: Could you explain this change?  (And the corresponding change for aioBoardHoverName.)

Comment (MikePinball) [reply PinscapeConfigTool/html/Configure.htm:6170]: I was trying something and forgot to change it back. The code should stay as
<code>var alias = outPortAlias[pin];</code>
for all three functions: mainBoardHoverName(), aioBoardHoverName(), and liteBoardHoverName()

Comment (mjrgh) [reply PinscapeConfigTool/html/Configure.htm:6170]: Great, I'll go ahead and roll it back.

Comment (MikePinball) [reply PinscapeConfigTool/html/Configure.htm:6170]: The binary download appears to be missing the four new images. Please refresh.

**Comment by mjrgh** [comment]
For future reference, that means you forgot to add them to the VS project.  What you need to do in the future when adding any new files to the HTML folder is this:

- Open the project in Visual Studio
- Go to the Solution Explorer pane
- Navigate to PinscapeConfigTool > html
- Right click on the **html** object and select Add > Existing Item...
- Select the new files
- They'll now appear in the html folder in the Solution Explorer tree; for each one:
  - Click on it to select it
  - In the Properties pane, change the value of **Copy to Output Directory** to **Copy if newer**

Make sure that when you add the files in the git tree, you add them to the **PinscapeConfigTool\html** folder.  **Don't** add them to the **bin** folder; those are always derived files.

If you want to create a new PR with the additions, I'll update from that.

**Comment by MikePinball** [comment]
I'm sorry I should have reread your notes from pull request #13. I remembered the issue with tabs and concentrated on making sure that wasn't a problem. And you obviously caught it last time as well. I would like to say third time is a charm but I don't forsee any other changes (except my private testing tool).

See pull request #16

**Comment by mjrgh** [comment]
No problem; I just added some notes about it to the repos (Developer Notes.txt) to make it easier for everyone (myself included) to remember the extra steps.

**Pull request merged**